### PR TITLE
Send abridged version of history with messages in group chats

### PR DESCRIPTION
**Pull request by cammellos**
When sending a message in a private group chat we send the whole history
for redundancy and allow out-of-order processing.
This can be very expensive in some chats, resulting in long delay when
sending a message and calculating the POW.
This commit improves the performance  by only forwarding the events
necessary for the user to be able to construct a group chat and process
correctly the message.

### CODE REVIEW

**Review by vkjr — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.68.0
+0.68.1
```
Comment (Samyoul) [inline VERSION]: I notice that your message caching also has the same version number. I don't know how much this matters,

File: protocol/v1/membership_update_message.go [hunk 2]
```diff
@@ -314,6 +314,78 @@ func (g Group) Events() []MembershipUpdateEvent {
...
+				events = append(events, event)
+			}
+		}
+
+	}
+
+	// Reverse events
+	for i, j := 0, len(events)-1; i < j; i, j = i+1, j-1 {
+		events[i], events[j] = events[j], events[i]
+	}
```
Comment (Samyoul) [inline protocol/v1/membership_update_message.go:384]: 😄 reminds me of algorithm practice.

**Review by Samyoul — APPROVED** [review]
Looks great.

**Comment by cammellos** [comment]
Closing as tested in a separate PR manually tested by QA, feedback has been addressed there

**Pull request closed without merging**